### DISCUSSION TITLE
Thank you for Soundy

### DISCUSSION THREAD

**Discussion by Kevin-N749** [syfds/soundy#47] [General]
Hi Syfds,
I just wanted to thank you for your work on Soundy.  
I am just starting to use Linux Mint, and I doubted greatly that I'd be able to control my Bose speakers with it.
I couldn't believe it when I found Soundy.
Soundy works flawlessly and I just wanted to tell you how happy I am with it.
-Kevin N.

**Comment by pracplayopen** [comment]
Also want to say thx- only have one windows machine so soundy makes it much simpler to control the speakers from anywhere.

Also confirm that multi-room zones work perfectly (binary/deb install), just as the official client (maybe even a little simpler).
